discretize/InnerProducts.py

```diff
@@ -71,14 +71,40 @@ def getEdgeInnerProduct(self, prop=None, invProp=False, invMat=False, doFast=Tru
         """
         return self._getInnerProduct('E', prop=prop, invProp=invProp, invMat=invMat, doFast=doFast)
 
+    def getNodalInnerProduct(self, prop=None, invProp=False, invMat=False, doFast=True):
+        """Generate the face inner product matrix
```

Comment:
```suggestion
        """Generate the nodal inner product matrix
```

Reply:
@dccowan, I am not sure that this is doing what is expected. I am playing around with tests and I think that this won't work out-of the box. If you are okay with it, let's postpone this to another pr. I can roll-back this commit and we can start a separate pr. Is that okay with you? 

(P.S. Once the review is started, it is usually better to start a new pr to add new functionality - otherwise little additions and tweaks can keep delaying things)